cachet.py

```diff
@@ -38,6 +38,25 @@ def getComponents(self):
 
         return self.__getRequest('/components')
 
+    def searchComponents(self, id=None, name=None, status=None, group_id=None, enabled=None, sort=None, order='desc', per_page=None):
+        '''
+        Search for Components using the advance api usage.
+
+        :return: :class:`Response <Response>` object
+        :rtype: requests.Response
+        '''
+
+        return self.__getRequest('/components?'+
+            ('id=%d'%id if id!=None else '')+
```

Comment:
Missing URL param separator `&`

Reply:
Well that's embarrassing...

I changed the get request to be able to pass parameters as dict in commit [c9852b8 ](https://github.com/imlonghao/cachet.python/pull/2/commits/c9852b8c0afec9374ff3cf17d7aa0556481b27f6)